1.7/administration/securing-your-cluster.md

```diff
@@ -1,6 +1,6 @@
 ---
-post_title: Security
-nav_title: Security
+post_title: Securing Your Cluster
+nav_title: Securing Your Cluster
```

Comment:
I was just looking through the dcos.io site and I noticed the first paragraph of this page https://dcos.io/docs/1.7/administration/id-and-access-mgt/configure-your-security/ seems redundant with your new page. Although your new page is much more detailed on comprehensive. As such, it would seem to supercede this lonely paragraph. Perhaps we could delete that first paragraph then, if you agree, to avoid the redundancy. That way there's no confusion.